src/Component/Editor/Constants.js

```diff
@@ -0,0 +1,18 @@
+const modules = {
+  toolbar: [
+    [{ header: [1, 2, false] }],
+    ["bold", "italic", "underline", "strike","blockquote"],  
+    
+  ],
+};
+
+const formats = [
+  "header",
+  "bold",
+  "italic",
+  "underline",
+  "strike",
+  "blockquote",
```

Comment:
is there any specific reason for putting the comma at the very last element!!
bcz as per what i know there's no need of a comma here after the last element!!
do correct me if i m wrong!!